scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -280,15 +282,16 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
 
 {% macro pinned_panel() %}
 <a name="pinned_variants"/>
-<div class="card border-info mb-3">
-  <div class="card-header">
+<div class="card border-info mb-3" style="border-width: 5px;">
+  <div class="card-header bg-info text-white">
     <strong>Pinned Variants</strong>
   </div>
   <div class="card-body">
     {% set duplicated_variants = [] %}
     {% if suspects_detailed %}
       {% for pinned in suspects_detailed|sort(attribute='variant_rank') %}
-        {% if pinned['_id'] not in causatives_detailed | map(attribute='_id') %}
+        {% if pinned['_id'] not in printed_vars %}
```

Comment:
We should try to move all of this logic so that it is done in the controllers instead of templates. But lets do that when we work on the performance stuff together